### DISCUSSION TITLE
make youtube ask before saving to playlist

### DISCUSSION THREAD

**Discussion by cr1spyapple** [PoomSmart/YTABConfig#30] [Q&A] [answered]
YouTube used to ask which playlist i want to use save a video to, but they removed that a while ago, is there anyway to change it back?

**Comment by CySxL** [comment, marked as answer]
> YouTube used to ask which playlist i want to use save a video to, but they removed that a while ago, is there anyway to change it back?

Same thoughts ! It's so annoying.

**Reply by qnblackcat** [reply]
Instead of tapping the Save button, just hold it. Problem solved!

**Reply by CySxL** [reply]
> Instead of tapping the Save button, just hold it. Problem solved!

Sir can I call you the savior ? 😂
How amazing that it does work !!

**Reply by cr1spyapple** [reply]
> Instead of tapping the Save button, just hold it. Problem solved!

Thank you so much!!!
